src/main/java/cloud/orbit/spring/OrbitActorsProperties.java

```diff
@@ -117,4 +118,14 @@ public Long getDeactivationTimeoutInMilliseconds() {
     public void setDeactivationTimeoutInMilliseconds(final Long deactivationTimeoutInMilliseconds) {
         this.deactivationTimeoutInMilliseconds = deactivationTimeoutInMilliseconds;
     }
+
+    public Integer getExecutionPoolSize()
```

Comment:
Yep. There has also been some discussion around parts of Orbit using Scala/Kotlin moving forward which have tools to avoid this kind of thing.